src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -0,0 +1,256 @@
+import logging
```

Comment:
The extra/overriding endpoints from `src/fides/api/ops/api/v1/endpoints/user_endpoints.py` could be moved into here if they were extra, or removed if overriding.